### PR TITLE
chore(docs): Updated readme with instructions.

### PR DESCRIPTION
**Pull request by bee344**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,25 @@ tests derived from Smart Contracts (such as Uniswap V3) and lower level testing
...
 ```sh
-~ ./init.sh
+~ ./init.sh <CHAIN> <TEST>
 ```
-This will install the necessary packages and run the tests in order.
+Currently the available options for `CHAIN` to test against are:
+* `--acala`
+* `--moonbeam`
+* `--astar`
+* `--ehtereum`
```
Comment (marshacb) [inline README.md:18]:
```suggestion
* `--ethereum`
```

File: README.md [hunk 2]
```diff
@@ -9,6 +9,25 @@ tests derived from Smart Contracts (such as Uniswap V3) and lower level testing
...
+* `--polygon`
+
+With `--ethereum` being the default option.
+
+As for `TEST`, you can specify either `--mater-labs` to run the `matter-labs`
```
Comment (marshacb) [inline README.md:23]:
```suggestion
As for `TEST`, you can specify either `--matter-labs` to run the `matter-labs`
```

File: README.md [hunk 3]
```diff
@@ -9,6 +9,25 @@ tests derived from Smart Contracts (such as Uniswap V3) and lower level testing
...
+tests or `--smart-contracts` to run the test that deploy and check against the
```
Comment (marshacb) [inline README.md:24]:
```suggestion
tests or `--smart-contracts` to run the tests that deploy and check against the
```

**Review by marshacb — APPROVED** [review]
just a couple of nits. LGTM

**Comment by bee344** [comment]
Closed in favor of #3

**Pull request closed without merging**